tests/compas/geometry/test_core.py

```diff
@@ -202,3 +203,14 @@ def test_area_triangle(triangle, R):
     assert close(area_triangle(triangle.points), 0.5)
     assert close(area_triangle_xy(triangle.points), 0.0)
     assert close(triangle.area, 0.5)
+
+
+# ==============================================================================
+# normals
+# ==============================================================================\
+
+def test_normal_polygon():
+    polygon = [(0, 0, 0), (10, 0, 0), (10, 10, 0), (0, 10, 0)]
+    normal = normal_polygon(polygon, unitized=False)
+    area = area_polygon(polygon)
+    assert length_vector(normal) == area
```

Comment:
this could fail due to floating point precision. please use `assert close(area, length_vector(normal))`.
a sanity check would also be good `assert close(area, 100.0)`